src/test/kotlin/org/jetbrains/intellij/platform/gradle/tasks/PrepareSandboxTaskTest.kt

```diff
@@ -361,6 +362,59 @@ class PrepareSandboxTaskTest : IntelliJPluginTestBase() {
         )
     }
 
+    @Test
+    fun `prepare sandbox for splitMode with plugin installed on frontend`() {
```

Comment:
BTW if tests are written on JUnit5, I would simplify this code by using a parameterized test.

Reply:
Good point — I'll move toward JUnit5 soon.